runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Scheduler/Scheduler.thunks.ts

```diff
@@ -15,7 +15,7 @@ export const getScheduledJobs = createAsyncThunk<ScheduledJob[]>(
 
 export const getActiveJobs = createAsyncThunk<IProcessInstance[]>(
     'scheduledJobs/getActiveJobs',
-    () => axios.get(`/api/process-instances?status=${processStatuses.toString()}`)
+    () => axios.get(`/api/process-instances?status.in=${processStatuses.toString()}`)
```

Comment:
is it a convention to use '.'?

Reply:
yes, I think we have always used the `.` convention to search by criteria